### PR TITLE
Unit tests jest

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-07-24 at 2 00 04 PM

### CODE REVIEW

File: unit-tests-jest/numbers.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Modifies an object by multiplying the value of each key, but only if that
+ * value is a number. Key names are arbitrary.
+ * @param {Object} obj The object to modify.
+ * @param {number} multiplier The number to multiply by.
+ * @returns the input object.
+ */
+export function multiplyBy(obj, multiplier) {
+  const result = {};
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.js:37]: This makes a copy and does not modify the original, but spec says "Modifies an object".

File: unit-tests-jest/numbers.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.js:2]: These are good happy path tests, but it would be good to also test the edge cases.

**Review by RobertGardner — APPROVED** [review]
